src/ucs/datastruct/usage_tracker.h

```diff
@@ -13,29 +13,29 @@
 #include <ucs/datastruct/linear_func.h>
 #include <ucs/datastruct/lru.h>
 
-/* Usage Tracker element data structure */
+/* Usage Tracker element data structure. */

```

Comment:
do we use trailing comma in other comments (for example, LRU)? pls make it consistent.

Reply:
in LRU we use it as well

Reply:
you mean adding '.' in LRU comments?

Reply:
ok

Reply:
added in LRU

Reply:
the "ok" was reply to "in LRU we use it as well".
but now i see we **don't** use it in LRU.
So, we should not use it in usage_tracker, too.

Reply:
we use it on comments over functions for example, but not comments over fields. 
so I'm confused, are you talking specifically about the field comments?

Reply:
> are you talking specifically about the field comments?
Yes